portfolio/src/main/webapp/script.js

```diff
@@ -25,3 +25,25 @@ setInterval(() => {
   }
 }, speed);
 
+function getGreeting() {
+  console.log('Fetching greeting');
+
+  const greetingPromise = fetch('/data');
+
+  greetingPromise.then(responseHandler);
+}
+
+function responseHandler(response) {
+  console.log('Handling response');
+
+  const textPromise = response.text();
+
+  textPromise.then(addGreetingToDom);
+}
+
+function addGreetingToDom(greeting) {
```

Comment:
Try to document the parameters. It will help you understand the code when you come back later. 
https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
Added, thank you!